update/plan/update_230000.go

```diff
@@ -0,0 +1,269 @@
+		types.NewLazySchedule(
+			genesisTime.AddDate(0, 13, 0).Unix(), genesisTime.AddDate(0, 17, 0).Unix(), sdk.NewDecWithPrec(40, 2)),
+		types.NewLazySchedule(
+			genesisTime.AddDate(0, 17, 0).Unix(), genesisTime.AddDate(0, 18, 0).Unix(), sdk.NewDecWithPrec(15, 2)),
+	)
+
+	lazyVestingSchedule = types.NewLazyVestingSchedule(assets.MicroLunaDenom, lazySchedules)
+	lazyVestingSchedules = append(lazyVestingSchedules, lazyVestingSchedule)
+
+	return lazyVestingSchedules
+}
+
+// only terra1y9n2ywyu5dahtxar6k4z4jz97ynt8km4catuk6 account has difference vesting schedule
+// except that all seed account does not have any other schedules
+func updateSeedSchedules(gvacc types.GradedVestingAccount) []types.LazyVestingSchedule {
+	vestingSchedules := gvacc.GetVestingSchedules()
+	if len(vestingSchedules) != 1 || vestingSchedules[0].GetDenom() != assets.MicroLunaDenom {
+		panic("Invalid Preseed Account")
+	}
+
+	vestingSchedule := vestingSchedules[0]
+	ratio := sdk.OneDec()
+
+	fmt.Println(gvacc.GetAddress().String())
+	// strict preseed account check
+	if gvacc.GetAddress().String() == "terra1y9n2ywyu5dahtxar6k4z4jz97ynt8km4catuk6" {
+		ratio = sdk.NewDecWithPrec(467, 3)
+
+		if len(vestingSchedule.Schedules) != 5 {
+			panic("Invalid Preseed Account")
+		}
+
+		for _, s := range vestingSchedule.Schedules {
+			if s.GetCliff() == 1558677600 && s.GetRatio().Equal(sdk.NewDecWithPrec(47, 3)) {
+				continue
+			} else if s.GetCliff() == 1561356000 && s.GetRatio().Equal(sdk.NewDecWithPrec(47, 3)) {
+				continue
+			} else if s.GetCliff() == 1563948000 && s.GetRatio().Equal(sdk.NewDecWithPrec(47, 3)) {
+				continue
+			} else if s.GetCliff() == 1582524000 && s.GetRatio().Equal(sdk.NewDecWithPrec(326, 3)) {
+				continue
+			} else if s.GetCliff() == 1603519200 && s.GetRatio().Equal(sdk.NewDecWithPrec(533, 3)) {
+				continue
+			} else {
+				panic("Invalid Pressed Account")
+			}
+		}
+	} else {
+		if len(vestingSchedule.Schedules) != 4 {
+			panic("Invalid Preseed Account")
+		}
+
+		for _, s := range vestingSchedule.Schedules {
+			if s.GetCliff() == 1558677600 && s.GetRatio().Equal(sdk.NewDecWithPrec(10, 2)) {
+				continue
+			} else if s.GetCliff() == 1561356000 && s.GetRatio().Equal(sdk.NewDecWithPrec(10, 2)) {
+				continue
+			} else if s.GetCliff() == 1563948000 && s.GetRatio().Equal(sdk.NewDecWithPrec(10, 2)) {
+				continue
+			} else if s.GetCliff() == 1582524000 && s.GetRatio().Equal(sdk.NewDecWithPrec(70, 2)) {
+				continue
+			} else {
+				panic("Invalid Pressed Account")
+			}
+		}
+	}
+
+	var lazyVestingSchedules []types.LazyVestingSchedule
+	var lazyVestingSchedule types.LazyVestingSchedule
+	var lazySchedules []types.LazySchedule
+
+	genesisTime := time.Unix(genesisUnixTime, 0)
+	lazySchedules = append(lazySchedules,
+		types.NewLazySchedule(
+			genesisTime.AddDate(0, 1, 0).Unix(), genesisTime.AddDate(0, 2, 0).Unix(), ratio.Mul(sdk.NewDecWithPrec(10, 2))),
+		types.NewLazySchedule(
```

Comment:
Think even distribution from month 2 to month 10 should be fine

Reply:
I think so too. I have updated code and test please check